redirectionio/proxy-symfony/0.1/config/packages/redirectionio.yaml

```diff
@@ -1,5 +1,6 @@
 # Documentation: https://github.com/redirectionio/proxy-symfony
 redirection_io:
+    project_key: FIXME
```

Comment:
Could it be an env var? That would be more consistent with how we approach the same situation for other projects.

Reply:
I don't know, but we are using the same project_key for a project. This value will be static (it's the same in dev / preprod / prod). The end user configure each instances via the website.

So adding a new env var will add an unnecessary complexity to me.

WDYT?